Added information for QEMU Guest Agent activation (#1048)
MichaIng/DietPi-Docs: docs/install.md

        Click **Next**.
    1. Tab **System**: As **Machine** you can select `q35`, but the older default will work as well. We recommend the para-virtualised **VirtIO SCSI** controller, which should be the default.  
        _**Important:** The check box `Qemu Agent` should to be activated._  
        Click **Next**.

        ??? info "Why to activate the `Qemu Agent` check box?"
            The QEMU Guest Agent (QGA) is a service that runs inside the virtual machine (VM) and enables better communication between the host (`Proxmox VE`) and the guest operating system:

            - **Accurate Guest Information**: QGA lets `Proxmox VE` fetch real-time IP addresses and guest details, providing a more seamless management experience.
            - **Consistent Backups**: Using the QGA commands `fs-freeze` and `fs-thaw` helps to ensure that the guest’s filesystem is in a consistent state before snapshots or backups are made, therefore reducing the risk of data corruption.
            - **Automated TRIM**: When moving or migrating VMs on thin-provisioned storage, QGA triggers TRIM commands to reclaim unused disk space, optimizing storage usage.
            

>             The QEMU Guest Agent is installed by DietPi during the initial first run setup. Therefore only the check box needs to be activated during the creation of the VM.

Reviewer: ```suggestion
            The QEMU Guest Agent is installed automatically by DietPi during the initial first run setup, if the check box has been activated before the first startup of the VM.
```
